.github/workflows/build.yml

```diff
@@ -1,18 +1,14 @@
 name: build radicale
```

Comment:
rename to radicale.yml to go with

- calcurse.yml
- projects.yml
- features.yml

Reply:
The container also needs to be updatable using yq ala create_job.yml